Maestro/subscriptions.json

```diff
@@ -906,6 +906,18 @@
           "PB_VersionsRepoRef": "<trigger-commit>"
         }
       }
+    },
+    {
```

Comment:
I'd like it if there were a comment header here for the 2.1-preview1 dir, maybe even mentioning that the publish steps are intentionally not included, just verification.